app/scripts/modules/amazon/src/loadBalancer/configure/application/ALBListeners.tsx

```diff
@@ -233,9 +305,40 @@ class ALBListenersImpl extends React.Component<
     this.updateListeners();
   };
 
+  private configureOidcClient = (action: IListenerAction): void => {
+    ConfigureOidcConfigModal.show({ config: action.authenticateOidcConfig }).then((config: any) => {
+      action.authenticateOidcConfig = config;
+    });
+  };
+
+  private removeAuthAction(actions: IListenerAction[], authIndex: number, ruleIndex: number): void {
+    const removedAuthAction = actions.splice(authIndex, 1)[0];
+    this.removedAuthActions[ruleIndex || -1] = removedAuthAction;
```

Comment:
what is `-1` for?

Reply:
-1 is for default actions, since they don't follow the same format, again, because aws